narratives/ncov_sit-rep_2020-03-20.md

```diff
@@ -0,0 +1,2453 @@
+---
+<!-- Translators: Only text after : in the above ^ needs to be translated -->
+<!-- Comment tags like these do not need to be translated, they are only to help you! -->
+<!-- Ensure that links always end in a 'letter' (. counts) If some kind of text doesn't follow them, it breaks the slide. -->
+<!-- numbers can be tagged ilke this: 161</tag> - this is just for us to help find them to update! Just leave in the </tag> bit. -->
+
+<!-- This is left-side text 1 -->
+# [Table of Contents](https://nextstrain.org/ncov/2020-03-19?d=tree,map&p=grid)
+
+* [Background resources](https://nextstrain.org/community/narratives/nextstrain/ncov@ncov_sit-rep_2020-03-21/sit-rep_2020-03-20?n=2).     
+* [A note about sampling](https://nextstrain.org/community/narratives/nextstrain/ncov@ncov_sit-rep_2020-03-21/sit-rep_2020-03-20?n=3).
+* [Sequence data & travel histories](https://nextstrain.org/community/narratives/nextstrain/ncov@ncov_sit-rep_2020-03-21/sit-rep_2020-03-20?n=5).
+* [Introduction of COVID-19 to almost all countries](https://nextstrain.org/community/narratives/nextstrain/ncov@ncov_sit-rep_2020-03-21/sit-rep_2020-03-20?n=7).
+* [How outbreaks grow](https://nextstrain.org/community/narratives/nextstrain/ncov@ncov_sit-rep_2020-03-21/sit-rep_2020-03-20?n=8).
+* [How the Washington outbreak has grown](https://nextstrain.org/community/narratives/nextstrain/ncov@ncov_sit-rep_2020-03-21/sit-rep_2020-03-20?n=9).
+* [How case counts change with social distancing](https://nextstrain.org/community/narratives/nextstrain/ncov@ncov_sit-rep_2020-03-21/sit-rep_2020-03-20?n=10).
+* [What you can do](https://nextstrain.org/community/narratives/nextstrain/ncov@ncov_sit-rep_2020-03-21/sit-rep_2020-03-20?n=11).
+* [FAQ & common misconceptions](https://nextstrain.org/community/narratives/nextstrain/ncov@ncov_sit-rep_2020-03-21/sit-rep_2020-03-20?n=12).
+* [Scientific credit](https://nextstrain.org/community/narratives/nextstrain/ncov@ncov_sit-rep_2020-03-21/sit-rep_2020-03-20?n=13).
+
+<!-- This is right-side text -->
+```auspiceMainDisplayMarkdown
+# Executive summary
+
+Here, we analyzed 713 publicly shared COVID-19 genomes. By comparing these viral genomes to each other, we can characterize how COVID-19 is evolving and moving around the world.
+
+In this report, we highlight that the virus has been introduced and is circulating locally in many parts of the globe. This emphasizes the importance of rapid, widespread testing, as well as a strong focus on slowing local transmissions.
+
+In this week's updates, we report:  
+
+* Evidence for travel-related introductions to many parts of the world.  
+* Many places with recent introductions can expect to see cases expand in 4-8 week timeframe. They should prepare now.   
+* Many places that have recently implemented social distancing measures can expect to see cases continue to accumulate in the short-term, but for new cases to plateau and decline in the medium- to long-term.
```

Comment:
For clarity's sake - 
Replace: "...but for new cases to plateau and decline in the medium- to long-term."
With: "...but new cases will likely plateau and decline in the medium- to long-term."